### ISSUE TITLE
DJ not bobbing / dancing

### ISSUE THREAD

**Issue by turntablefm** [TTFM-Labs/boring-infrastructure#439]
-Something that I loved from these kinds of apps is the head bobbing/nodding along. I saw that reflected in the audience (with my many, many self-duplicates of mine) from entering the room over and over. What I did not see it with was the DJ. Often times if you can see your performer feeling it, you're going to start feeling it too eventually. Plus, when you're an artist up here playing your music, your text engagement as well as character involvement is what excites listeners even more to be a part of the experience.

**Comment by kowalski** [comment]
ah, dj should dance too? I didn't know, that's doable I think :)

**Comment by turntablettfm** [comment]
Yes definitely I think it’s in a video. I think it can be a full
Body up and down
